src/lib/utils/formatter/token.ts

```diff
@@ -64,7 +76,7 @@ export const formatUTokenWithPrecision = (
   return formatDecimal({
     decimalPoints: decimalPoints || precision,
```

Comment:
Consider using nullish coalescing (??) instead of logical OR (||). This is because when we intentionally set `decimalPoints` to 0, it defaults to using `precision` instead.